File: 1938-minimum-operations-to-make-the-array-increasing/minimum-operations-to-make-the-array-increasing.py
class Solution(object):
    def minOperations(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        c=0
        for i in range(1,len(nums)):
            if nums[i-1]>=nums[i]:
                adding=nums[i-1]-nums[i]+1
                nums[i]+=adding
                c+=adding
        return c
        

File: 1938-minimum-operations-to-make-the-array-increasing/minimum-operations-to-make-the-array-increasing.java
class Solution {
    public int minOperations(int[] nums) {
        int c=0;
        for(int i=1;i<nums.length;i++){
            if(nums[i-1]>=nums[i]){
                int adding=nums[i-1]-nums[i]+1;
                nums[i]+=adding;
                c+=adding;
            }
        }
        return c;
    }
}